OxfordBodleian/Juel-Jensen/BDLaethd17.xml

```diff
@@ -33,87 +37,309 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    ለዓለመ፡ ዓለም፡ አሜን። ወትብል፡ ከመዝ፡ ኃሠሥኩከ፡ <cb n="b"/>
+                                    <hi rend="rubric">እማዕምቀ፡ ልብየ። ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡</hi> ርድአኒ፡
+                                    አንተ፡ ውእቱ፡ ረዳኢ ፡ ወፍትሐኒ፡ <hi rend="rubric">እማዕሥረ፡ ኃጢአት። ኦእግዚእየ፡
+                                        ኢየሱስ፡ ክርስቶስ፡</hi> ርድአኒ፡ አንተ፡ ውእቱ፡ ረዳኢ፡ ለአድኅኖትነ፡ ኂሩትከ፡ ይርከበኒ፡
+                                    ፍጡነ፡ ወጸልለኒ፡ ታሕተ፡ ጽላሎተ፡ ክነፊከ፡ በሰዱስ፡ <pb n="109v"/>
+                                    <cb n="a"/> ዕለት፡ ፈጠርከነ፡ ኵሎ፡ ዓለመ። </incipit>
+                                <explicit xml:lang="gez">
+                                    <pb n="123r"/><cb n="b"/>ብሩክ፡ ውእቱ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ሥሉስ፡
+                                    ፍጹም። ንስግድ፡ ሎቱ፡ ወንሰብሖ፡ ሎቱ፡ ይደሉ፡ ስብሐት፡ እምአፈ፡ መላእክት፡ ወሰብእ፡ ወእምአፈ፡
+                                    ኵሉ፡ ፍጥረት፡ እለ፡ ሀለው፡ በምድር፡ ወበሰማያት። በባሕርኒ፡ ወበቀላያት፡ ለዓለመ፡ ዓለም፡ አሜን።
+                                </explicit>
+                            </msItem>
+
+                            <msItem xml:id="ms_i1.8">
+                                <locus from="123ra" to="139ra"/>
+                                <title type="complete" ref="LIT2505Weddas#Sunday" xml:lang="gez"/>
+                                <incipit xml:lang="gez">
+                                    <pb n="123r"/>
+                                    <cb n="a"/>
+                                    <hi rend="rubric">ጸሎተ፡ ቅዱስ፡ <persName ref="PRS3255CyrilI">ቄርሎስ፡
+                                        </persName>አቡ</hi>ነ፡ ሊቀ፡ ጳጳሳት፡ ዘሀገረ፡ <placeName
+                                        ref="LOC1338Alexan"> እለ፡ <hi rend="rubric"
+                                        >እስክንድርያ፡</hi></placeName>
+                                    <hi rend="rubric">ዘዕለተ፡ ሰን</hi>በት፡ ክርስቲያን፡ በረከቱ፡ ቅድስት፡ ትኩን፡ ምስለ፡
+                                    ገብሩ፡ <persName ref="PRS12973Rehr"><sic>ርኅሩኅ፡</sic></persName>
+                                    (sic) ለዓለመ፡ ዓለም። ስብሐት፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ። ቦሙ፡ አአምን፡ ወእትአመን፡
+                                    እስከ፡ ነፍስ፡ ደኃሪት። ስብሐት፡ ለከ፡ ኦእግዚአብሔር፡ <hi rend="rubric">ስብሐት።
+                                        አቄርብ፡ ለ</hi>ከ፡ በአማን፡ ዘይደሉ፡ ለዕበ<hi rend="rubric">የ፡
+                                        ቅድምናከ።</hi>
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <pb n="138v"/><cb n="b"/>እጽናዕ፡ በፍቅርከ። ወእትወከል፡ በብዝኃ፡ ምሕረትከ። በጸጋከ፡
+                                    ወሣህልከ፡ ኀቤከ፡ እስእል፡ እግዚኦ፡ ከመ፡ ትምርሃኒ፡ ፍኖተ፡ ዘነሐውር፡ ባቲ። እንተ፡ ይእቲ፡
+                                    ፍኖተ፡ ጻድቃን፡ ርትዕት። እስከ፡ <pb n="139r"/><cb n="a"/>
+                                    <sic>ከ፡</sic> ነፍስ፡ ደኃሪት፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን። </explicit>
+                            </msItem>
+
+                            <note>There is no explicit of the work (<ref type="work"
+                                    corresp="LIT2505Weddas#Explicit"/>) as given e. g., in GCA-005
```

Comment:
"as given, e.g.," with comma and no space. Please give IDs.

Reply:
corrected to ', e.g.'
Ids will be given as soon as the entries are created (I commented on this to not to forget).